### PR TITLE
Update demo features

### LINKED ISSUE #12796: Change demo fixture
**Describe the bug**

New demo fixtures 

https://docs.google.com/spreadsheets/d/1XFB8i66BV192E77fz5P0Wr6IWp0O0B_9E1ijB-fi06Y/edit?usp=sharing

Related to #11613

**Additionnal information**
PrestaShop version: N/A
PHP version: N/A

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | Change demo features
| Type?         | improvement 
| Category?     | IN
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes #12796
| How to test?  | See spreadsheet in ticket description

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13069)

### CODE REVIEW

File: install-dev/fixtures/fashion/data/feature_product.xml [hunk 1]
```diff
@@ -6,9 +6,27 @@
     <field name="id_feature_value"/>
   </fields>
   <entities>
-    <feature_product id="feature_product_6_9" id_feature="3" id_product="6" id_feature_value="9"/>
-    <feature_product id="feature_product_7_9" id_feature="3" id_product="7" id_feature_value="9"/>
-    <feature_product id="feature_product_8_9" id_feature="3" id_product="8" id_feature_value="9"/>
-    <feature_product id="feature_product_19_9" id_feature="3" id_product="19" id_feature_value="9"/>
+    <feature_product id="feature_product_1_4" id_feature="1" id_product="1" id_feature_value="4"/>
+    <feature_product id="feature_product_1_8" id_feature="2" id_product="1" id_feature_value="8"/>
+    <feature_product id="feature_product_4_6" id_feature="1" id_product="4" id_feature_value="6"/>
```
Comment (colinegin) [inline install-dev/fixtures/fashion/data/feature_product.xml:12]: add  <feature_product id="feature_product_3_6" id_feature="1" id_product="3" id_feature_value="6"/>

Comment (marionf) [reply install-dev/fixtures/fashion/data/feature_product.xml:12]: Done :)

**Review by PierreRambaud — DISMISSED** [review]

**Comment by marionf** [comment]
Install isn't working in Persian

capture d'écran_1241

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @marionf

**Pull request merged**